tests/test_irr_parse.py

```diff
@@ -17,10 +17,42 @@ def test_parse_basic(tmp_path):
     result_file = Path(context.out_dir_irr) / "irr_final.txt"
     assert result_file.exists()
 
-    content = []
     with open(str(result_file), 'r') as f:
-        for l in f.readlines():
-            content.append(l.strip())
-    assert content == ["193.254.30.0/24 AS12726", "212.166.64.0/19 AS12321", "212.80.191.0/24 AS12541"]
-    # removed duplicate network with higher ASN
-    assert "212.80.191.0/24 AS12542" not in content
+        content = [ l.strip() for l in f.readlines() ]
+    assert content == ["193.254.30.0/24 AS12726", "212.166.64.0/19 AS12321", "212.80.191.0/24 AS12541", "212.16.0.0/24 AS12346", "212.17.0.0/24 AS12347", "2345:2ca::/32 AS12345" ]
+
+
+def test_parse_validation_cases(tmp_path):
+    """
+    Test various IRR parsing validation cases:
+    - Timestamp based conflict resolution
```

Comment:
No need to list the test cases up here, we will always have to append to this list when adding something new and then maybe we forget at some point. The inline comments are good enough IMO.